opentracing-shim/include/opentelemetry/opentracingshim/propagation.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "opentelemetry/baggage/baggage.h"
+#include "opentelemetry/common/attribute_value.h"
+#include "opentelemetry/context/propagation/text_map_propagator.h"
+#include "opentelemetry/nostd/type_traits.h"
+#include "opentracing/propagation.h"
+#include "opentracing/value.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace opentracingshim
+{
+
+template<typename T, nostd::enable_if_t<std::is_base_of<opentracing::TextMapWriter, T>::value, bool> = true>
+class CarrierWriterShim : public opentelemetry::context::propagation::TextMapCarrier
+{
+public:
+  CarrierWriterShim(const T& writer) : writer_(writer) {}
+
+  // returns the value associated with the passed key.
+  virtual nostd::string_view Get(nostd::string_view key) const noexcept override
+  {
+    return "";
```

Comment:
Is returning empty value expected here?

Reply:
Yes, this carrier wraps around an Opentracing TextMapWriter, which is essentially an interface with a single Set method, so it should never require a Get method in the same situation. I'll add a comment for clarity.